packages/pipeline-editor/src/utils.ts

```diff
@@ -146,4 +146,23 @@ export default class Utils {
       this.deletePipelineAppdataField(node, currentFieldName);
     }
   }
+
+  /**
+   * Return the last portion (filename) of a path. The file extension is
+   * stripped if `excludeExtension` is set to true.
+   *
+   * @param path
+   * @param excludeExtension
+   */
+  static getFilenameFromPath(path: string, excludeExtension?: boolean): string {
```

Comment:
There's no need to add this function. It already exists in the `path` library, in fact we do exactly this in the `handleExportPipeline` function. Look at the following on line 802:
```
    const pipeline_name = path.basename(
      pipeline_path,
      path.extname(pipeline_path)
    );
```
this accomplishes the same task and doesn't require adding a new util

Reply:
ah ok, will remove

Reply:
done https://github.com/elyra-ai/elyra/pull/867/commits/4dfe3be0a09a5964f7d5e9c6b55e888571b72314